main.py

```diff
@@ -2650,5 +2651,32 @@ async def recieve_vote(request):
     await claim_reward(user, channeley, type)
     return web.Response(text="ok", status=200)
 
+@bot.tree.command(name="fact", description="a fact")
```

Comment:
description change is recommended
maybe use "get a random cat fact" or something